File: Bars.Gkh/libs/B4/model/suggestion/SugTypeProblem.js
Ext.define('B4.model.suggestion.SugTypeProblem', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'SugTypeProblem'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'RequestTemplate' },
        { name: 'Name' },
        { name: 'ResponceTemplate'},
        { name: 'Rubric'}
    ]
});

File: Bars.Gkh/Entities/Suggestion/SugTypeProblem.cs
using Bars.B4.DataAccess;

namespace Bars.Gkh.Entities.Suggestion
{
    /// <summary>
    /// Тип проблемы с шаблонами
    /// </summary>
    public class SugTypeProblem : BaseEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }
        /// <summary>
        /// Шаблон запроса
        /// </summary>
        public virtual string RequestTemplate { get; set; }
        /// <summary>
        /// Шаблон  ответа
        /// </summary>
        public virtual string ResponceTemplate { get; set; }
        /// <summary>
        /// Рубрика
        /// </summary>
        public virtual Rubric Rubric { get; set; }


    }
}